client/reader/start/controller.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import { Provider as ReduxProvider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import { trackPageLoad, setPageTitle } from 'reader/controller-helper';
+
+const analyticsPageTitle = 'Reader';
+
+export default {
```

Comment:
why an object as the default export instead of `export function start( context )...`?